### PR TITLE
display "not found" if poa info not found

### PR DESCRIPTION
**Pull request by amprokop**
Connects #2090 
and #2256 

Handle cases where BRIEFF.BFSO is missing, and display "Not Found" on the client if info is not found.

image

### CODE REVIEW

File: app/mappers/power_of_attorney_mapper.rb [hunk 1]
```diff
@@ -28,6 +28,13 @@ def get_rep_name_from_rep_record(rep_record)
 
   def get_poa_from_vacols_poa(vacols_code:, representative_record: nil)
     case
+    when vacols_code.blank?
+      # If vacols code is not present, look in the rep table for rep name,
+      # but leave rep type nil.
```
Comment (aroltsch) [inline app/mappers/power_of_attorney_mapper.rb]: Hey @prokop would you please explain in the comment why we are doing it.

File: client/app/certification/ConfirmCaseDetails.jsx [hunk 2]
```diff
@@ -167,12 +167,12 @@ export class ConfirmCaseDetails extends React.Component {
 
     let appellantInfoRowObjects = [
       {
-        vbms: bgsRepresentativeName,
-        vacols: vacolsRepresentativeName
+        vbms: bgsRepresentativeName || 'Representative Name Not Found',
```
Comment (aroltsch) [inline client/app/certification/ConfirmCaseDetails.jsx]: Do we want to capitalize each word? It looks a bit strange :)

Comment (amprokop) [inline client/app/certification/ConfirmCaseDetails.jsx]: ha. yes, you're right, will fix

File: app/mappers/power_of_attorney_mapper.rb (hunk 1 above)
Comment (amprokop) [inline app/mappers/power_of_attorney_mapper.rb]: @Rolsch sure

**Pull request merged**